base/xx-info

```diff
@@ -340,8 +340,12 @@ case "$1" in
     echo "XX_VENDOR=${XX_VENDOR}"
     if [ "$TARGETOS" = "linux" ]; then
       echo "XX_PKG_ARCH=${XX_PKG_ARCH}"
-      echo "XX_TRIPLE=${XX_TRIPLE}"
     fi
+    case "$TARGETOS" in
```

Comment:
I don't really understand why there is a `os` filter at all in here. Maybe it is a historic relic from the time only Linux was supported.

Reply:
Looks like this cond has been added here: https://github.com/tonistiigi/xx/pull/14/commits/1b606ea19fd72ca058cb750285e3bb3286b5baa9#diff-2e3d3a4d78f2753f9464b79e8ac808928efdaeb5e0976104bad6f360448ad856R304-R307

Maybe it was to handle specific suffixes:

https://github.com/tonistiigi/xx/blob/c6d32015eb3e03de264a54ea9237388b319118b1/base/xx-apt#L160-L167

...for mingw packages like the apt one?: 

https://github.com/tonistiigi/xx/blob/c6d32015eb3e03de264a54ea9237388b319118b1/base/xx-apt#L193-L208